odxtools/singleecujob.py

```diff
@@ -59,9 +63,15 @@ def from_et(et_element, doc_frags: List[OdxDocFragment]) -> "InputParam":
             oid=oid,
         )
 
-    def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
+    def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
+        return {}
```

Comment:
Sure this guy have no odx_id?

Reply:
I just looked it up again: `INPUT-PARAM` tags come with an optional `OID` attribute, but for some strange reason without an `ID`. (strangely enough, for `OUTPUT-PARAM` the `ID` attribute is mandatory. looks like an oversight in the spec to me...)